### PR TITLE
CORDA-1461 Fix smoke tests

### PR DESCRIPTION
**Pull request by tudor-malene**
The devMode changes were breaking the api tests.
This fixes them.

### CODE REVIEW

File: client/rpc/src/smoke-test/java/net/corda/java/rpc/StandaloneCordaRPCJavaClientTest.java [hunk 1]
```diff
@@ -48,7 +48,8 @@
             port.getAndIncrement(),
             port.getAndIncrement(),
             true,
-            Collections.singletonList(rpcUser)
+            Collections.singletonList(rpcUser),
+            true
```
Comment (vkolomeyko) [inline client/rpc/src/smoke-test/java/net/corda/java/rpc/StandaloneCordaRPCJavaClientTest.java]: Minor: I would use `devMode = true` to make it more explicit about purpose of boolean constant passed in.

Comment (tlil) [reply client/rpc/src/smoke-test/java/net/corda/java/rpc/StandaloneCordaRPCJavaClientTest.java]: +1

Comment (tudor-malene) [reply client/rpc/src/smoke-test/java/net/corda/java/rpc/StandaloneCordaRPCJavaClientTest.java]: This is a java class.

Comment (tlil) [reply client/rpc/src/smoke-test/java/net/corda/java/rpc/StandaloneCordaRPCJavaClientTest.java]: Ah 🙂 True that...

**Pull request merged**